worker/src/main/java/gov/cms/ab2d/worker/processor/ContractProcessorImpl.java

```diff
@@ -159,8 +159,10 @@ private void loadRequests(JobData jobData, Job job, Map<Long, CoverageSummary> p
             // Queue a patient
             CoverageSummary patient = patientEntries.next().getValue();
             assert job.getContract() != null;
+
             contractData.addEobRequestHandle(processPatient(contractData.getFhirVersion(),
                     patient, job.getContract(), jobData));
+            this.jobChannelService.sendUpdate(job.getJobUuid(), JobMeasure.PATIENT_REQUEST_QUEUED, 1);
```

Comment:
Not good, makes job tracking updates way too chatty.  We want to send updates in chunks.